src/documentation/domains/issue/controller/issue.controller.docs.ts

```diff
@@ -36,4 +36,34 @@
  *             application/json:
  *               schema:
  *                 $ref: "#/components/responses/ConflictException"
+ *   /api/issue/{issueId}/worked-time:
+ *     get:
+ *       summary: Retrieve worked seconds for a specific issue.
+ *       tags:
+ *         - "Issue"
+ *       security:
+ *         - bearerAuth: []
+ *       parameters:
+ *         - in: path
+ *           name: issueId
+ *           schema:
+ *             $ref: '#/components/parameters/IssueWorkedTimeParamsDTO'
+ *           required: true
+ *           description: The ID of the issue to retrieve worked seconds for
+ *       responses:
+ *         '200':
+ *           description: Issue worked seconds retrieved successfully
+ *           content:
+ *             application/json:
+ *               schema:
+ *                 type: number
```

Comment:
This should be a schema component. It's gonna be beneficial if we have to change this in the future to have it in another place.